src/plugins/intel_gpu/include/intel_gpu/primitives/pooling.hpp

```diff
@@ -88,6 +88,64 @@ struct pooling : public primitive_base<pooling> {
           with_output_size(true),
           output_size(output_size) {}
 
+    /// @brief Constructs pooling primitive.
+    /// @param id This primitive id.
+    /// @param input Input primitive id.
+    /// @param mode Pooling mode.
+    /// @param stride Defines shift in input buffer between adjacent calculations of output values.
+    /// @param size Pooling kernel size.
+    /// @param pad Defines logical pad value added to input tensor.
+    pooling(const primitive_id& id,
```

Comment:
It looks like you can add `dilation` to existing constructors with default values
There is no need to create new ones